requirements.txt

```diff
@@ -1,2 +1,3 @@
 numpy
 protobuf>=3.20.2
+google-re2
```

Comment:
I would make this dependency optional: `pip install onnx[text]`.

Reply:
Have done so.